python/getting_started.py

```diff
@@ -0,0 +1,133 @@
+        canonical_request += canonical_headers + '\n'
+        canonical_request += signed_headers + '\n'
+        canonical_request += payload_hash
+
+        credential_scope = datestamp + '/' + self.__aws_region + '/' + self.__aws_service + '/' + 'aws4_request'
+        string_to_sign = self.__ALGORITHM + '\n'
+        string_to_sign += amzdate + '\n'
+        string_to_sign += credential_scope + '\n'
+        string_to_sign += hashlib.sha256(canonical_request.encode('utf-8')).hexdigest()
+
+        signing_key = self.__get_signature_key(
+            self.__aws_secret_access_key, datestamp, self.__aws_region, self.__aws_service)
+
+        signature = hmac.new(
+            signing_key, string_to_sign.encode('utf-8'), hashlib.sha256).hexdigest()
+
+        authorization_header = self.__ALGORITHM + ' '
+        authorization_header += 'Credential=' + self.__aws_access_key_id + '/' + credential_scope + ', '
+        authorization_header += 'SignedHeaders=' + signed_headers + ', '
+        authorization_header += 'Signature=' + signature
+
+        headers = {
+            'x-amz-date': amzdate,
+            'Authorization': authorization_header,
+        }
+
+        return headers
+
+
+class RequestHandler(object):
+    def __init__(self, endpoint, aws_signing_v4):
+        self.__endpoint = endpoint
+        self.__aws_signing_v4 = aws_signing_v4
+
+    def __to_canonical_querystring(self, params):
+        canonical_querystring = ''
+        # parameters have to be sorted alphabetically for the signing part
+        for param_key, param_value in sorted(params.iteritems()):
+            if canonical_querystring != '':
+                canonical_querystring += '&'
+            canonical_querystring += param_key + '=' + urllib.quote(param_value)
+        return canonical_querystring
+
+    def get_request(self, path, params, api_token=None):
+        canonical_querystring = self.__to_canonical_querystring(params)
+        headers = self.__aws_signing_v4.headers_for_get_method(path, canonical_querystring)
+
+        # The 'host' header is added automatically by the Python 'requests' library.
+        headers['Accept'] = 'application/json'
+        headers['Content-type'] = 'application/json'
+        headers['x-api-key'] = api_key
+
+        # All endpoints require the API token, except the API token endpoint.
+        if api_token:
+            headers['x-dnbapi-jwt'] = api_token
+
+        request_url = self.__endpoint + path + '?' + canonical_querystring
+        return requests.get(request_url, headers=headers)
+
+
+if __name__ == '__main__':
+    # Developer's credentials
+    client_id = ''
+    client_secret = ''
+    api_key = ''
+
+    # AWS signing v4 constants
+    aws_host = 'developer-api-sandbox.dnb.no'
+    aws_region = 'eu-west-1'
+    aws_service = 'execute-api'
+
+    openbanking_endpoint = 'https://developer-api-sandbox.dnb.no'
+
+    aws_signing_v4 = AwsSigningV4(client_id, client_secret, aws_host, aws_region, aws_service)
```

Comment:
Maybe use keyword arguments so people wont accidentally switch them up when creating a AwsSigningV4 object?

Reply:
Added keyword arguments and improved the coding style